onnx/defs/math/defs.cc

```diff
@@ -259,19 +259,17 @@ and output blobs.
                "computed element-wise");
 
 OPERATOR_SCHEMA(Pow)
-    .NumInputs(1)
+    .NumInputs(2)
     .NumOutputs(1)
-    .Attr("exponent",
```

Comment:
I would prefer keeping this attribute. Frontends could already know the exponent and it will be inconvenient (also inefficient) to always create an extra input tensor).

Reply:
The problem is we have case in which the exponent can change, specially for custom learner. We added exponent as input in order to support Keras backend and other scenario. Atteribute will be limiting in this case.

Reply:
I'm not against adding exponent as Input for dynamic cases, I'm saying let's keep the attribute for static cases.

Reply:
@bddppq  what the advantage? static case could use the argument also? What if the user specify both?

Reply:
Copy from my first comment:

> Frontends could already know the exponent and it will be inconvenient (also inefficient) to always create an extra input tensor).

In caffe2 we (will) disallow user specifying both.